client/src/components/Toasts/Toast.js

```diff
@@ -47,11 +47,12 @@ const Toast = ({ type, text, onDismiss, dismissed, actions }) => {
 
 Toast.propTypes = {
   ...toastShape,
-  onDismiss: PropTypes.func.isRequired
+  onDismiss: PropTypes.func
 };
 
 Toast.defaultProps = {
-  actions: []
+  actions: [],
+  onDismiss: () => {},
```

Comment:
This is to fix an unrelated console warning after running `yarn dev`

`Warning: Failed prop type: The prop `toasts[0].onDismiss` is marked as required in `Toasts`, but its value is `undefined`.`